Makefile

```diff
@@ -168,7 +168,7 @@ test-fuzz:
 ## test-interchain: Run interchain tests in verbose mode. Requires Docker.
 test-interchain:
 	@echo "--> Running interchain tests"
-	@go test ./test/interchain -v
+	@cd ./test/interchain && go test . -v
```

Comment:
do we need the "." ?

Reply:
- `go test .` runs only tests in the current directory, 
- `go test` runs tests in the current package directory, but if run from a module root, it will run tests in all packages in the module.

In this scenario, there's only one test in that directory and both commands run it.